src/formatDate.js

```diff
@@ -8,7 +8,44 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const INPUT_SYMBOL = fromFormat.pop();
+  const OUTPUT_SYMBOL = toFormat.pop();
```

Comment:
Similarly to the previous comment, use a `const` to access the last element of `toFormat` array instead of `pop()` to prevent modifying the original array.